packages/compass-query-bar/src/components/generative-ai/ai-text-input.tsx

```diff
@@ -261,15 +263,13 @@ function AITextInput({
 const ConnectedAITextInput = connect(
   (state: RootState) => {
     return {
-      isFetching:
-        state.aiQuery.aiQueryAbortController &&
-        !state.aiQuery.aiQueryAbortController.signal.aborted,
-      didSucceed: state.aiQuery.didSucceed,
+      isFetching: state.aiQuery.aiQueryFetchId !== -1,
```

Comment:
Super nit, this is totally valid code in this case, but I guess what you actually care here about is the `status`, and not whether or not fetch id is in the state or not, so might make more sense to derive it from it? 🤔

Reply:
Definitely what I meant to do! Thanks for catching!